### ISSUE TITLE
ArgumentOutOfRangeException for TransformGestureBase.getPointScreenPosition

### ISSUE THREAD

**Issue by cuibonobo** [TouchScript/TouchScript#243]
We just deployed an app to the Google Play store that uses TouchScript (thank you!) and I noticed an odd exception in my analytics:

```
ArgumentOutOfRangeException: Argument is out of range.
Parameter name: index

System.Collections.Generic.List`1[TouchScript.TouchPoint].get_Item (Int32 index)
TouchScript.Gestures.Base.TransformGestureBase.getPointScreenPosition (Int32 index)
TouchScript.Gestures.Base.TransformGestureBase.touchesMoved (IList`1 touches)
TouchScript.Gestures.Gesture.INTERNAL_TouchesMoved (IList`1 touches)
TouchScript.GestureManagerInstance.doUpdateMoved (TouchScript.Gestures.Gesture gesture, IList`1 touches)
TouchScript.GestureManagerInstance.update (IList`1 touches, System.Action`1 process, System.Action`2 dispatch)
TouchScript.GestureManagerInstance.touchesMovedHandler (System.Object sender, TouchScript.TouchEventArgs touchEventArgs)
TouchScript.Utils.EventHandlerExtensions.InvokeHandleExceptions[TouchEventArgs] (System.EventHandler`1 handler, System.Object sender, TouchScript.TouchEventArgs args)
UnityEngine.Debug:LogException(Exception)
TouchScript.Utils.EventHandlerExtensions:InvokeHandleExceptions(EventHandler`1, Object, TouchEventArgs)
TouchScript.TouchManagerInstance:updateUpdated(List`1)
TouchScript.TouchManagerInstance:updateTouches()
TouchScript.TouchManagerInstance:Update()
```

This exception is very rare so I'm not worried about it (I only have 14 reports of this error against 50K+ installs), but I thought it would be good to note. The only device I have listed that reported this error is `Android OS 6.0.1 / API-23 (MMB29M/20170105.161203)`.

**Comment by valyard** [comment]
Thanks. I'll examine how this could happen.

**Comment by valyard** [comment]
I think I accidentally fixed this in 9.0. I just can't see how it can happen from the code.
